content/cybersecurity/concepts/nmap/terms/ping-sweep/ping-sweep.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: 'Ping Sweep'
+Description: 'A network scanning technique to identify active devices on a network by pinging a range of IP addresses.'
+Subjects:
+  - 'Computer Science'
+  - 'Cybersecurity'
+  - 'Networking'
+Tags:
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+  - 'Networking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/scan-systems-with-nmap'
+---
+
+A **ping-sweep** is a network scanning technique to identify active devices on a network by pinging a range of IP addresses. It is specified with the `-sn` option.  Commpared to other methods, ping sweeps can be harder to detect as it is not as aggressive and can skip regular scan stages, making it more of an advantage.
+
+## Syntax
+
+```pseudo
+nmap -sn <subnet>/<mask>
+```
+
+The above identifies all of the IP addresses that are currently online without sending any packets to these hosts.
+
+## Example
+
+The following example runs a ping-sweep on `192.168.0.1/24`:
```

Comment:
Can we refer to subnet IP and mask as the host rather than the specific example? cc: @SSwiniarski